src/frontend/src/optimizer/plan_node/logical_scan.rs

```diff
@@ -181,6 +181,75 @@ impl LogicalScan {
             .collect()
     }
 
+    /// Return indexes can satisfy the required order.
+    /// The `prefix` refers to optionally matching columns of the index
+    /// It is unordered initially.
+    /// If any are used, we will return the fixed `Order` prefix.
+    pub fn indexes_satisfy_order_with_prefix(
```

Comment:
We can reimplement the `indexes_satisfy_order` method by leveraging the `indexes_satisfy_order_with_prefix` implementation.

Reply:
fixed